chapters/publication.tex

```diff
@@ -492,30 +485,31 @@ \subsection{Releasing a replication package}
 (However, there is a strict size restriction of 100MB per file and
 a restriction on the size of the repository as a whole,
 so larger projects will need alternative solutions.)
-
 However, GitHub is not ideal for other reasons.
```

Comment:
Suggestion:

However, GitHub is not the ideal platform to release reproducibility packages.
It is built to version control code, and to facilitate collaboration on it.
Features to look for in a platform to release such packages and that are not offered by GitHub, include:
the possibility to store data and documentation as well as code, 
the creation of a static copy of its content, that cannot be changed or removed,
and the assignment of a permanent digital object identifier (DOI) link.

Reply:
changed to your suggestion in 09b05010f5d0473fbcec302d63493cdf38c52987